src/mmw/js/src/core/views.js

```diff
@@ -167,12 +167,65 @@ var MapView = Marionette.ItemView.extend({
 
         this._leafletMap.invalidateSize();
 
-        if (this.model.get('areaOfInterest')) {
-            this._leafletMap.fitBounds(this._areaOfInterestLayer.getBounds());
+        var areaOfInterest = this.model.get('areaOfInterest');
+        if (areaOfInterest) {
+            var layer = new L.GeoJSON(areaOfInterest);
+            this._leafletMap.fitBounds(layer.getBounds(), { reset: true });
         }
     }
 });
 
+// Apply a mask over the entire map excluding bounds/shape specified.
+function applyMask(featureGroup, shapeLayer) {
+    var worldBounds = L.latLngBounds([-90, -360], [90, 360]),
```

Comment:
To the best of my knowledge, latitude ranges from -180 to +180 degrees.  Is there some special reason for the double wrap?